src/main/scala/rps/model/Move.scala

```diff
@@ -1,25 +1,18 @@
 package rps.model
 
-import io.buildo.enumero.annotations.indexedEnum
-import io.buildo.enumero.{CaseEnumIndex, CaseEnumSerialization}
+import io.buildo.enumero.annotations.enum
+import io.buildo.enumero.{CaseEnumSerialization}
 
-@indexedEnum trait Move {
-  type Index = String
-  Rock { "0" }
-  Paper { "1" }
-  Scissors { "2" }
+@enum trait Move {
+  Rock
+  Paper
+  Scissors
 }
 
 object MoveHelpers {
   implicit class ConversionHelper(x: String) {
     def convertToMove(): Option[Move] = {
-      CaseEnumIndex[Move].caseFromIndex(x)
-    }
-  }
-
-  implicit class ReverseConversionHelper(x: Move) {
-    def convertToString(): String = {
-      CaseEnumSerialization[Move].caseToString(x)
+      CaseEnumSerialization[Move].caseFromString(x)
```

Comment:
IIRC you can do `Move.caseFromString(x)`

Reply:
Also, you're not using `convertToString` anymore, right?

Reply:
> Also, you're not using convertToString anymore, right?

didn't realize I don't :)